grails-app/assets/javascripts/services/player-service.js

```diff
@@ -44,7 +47,7 @@ streamaApp.factory('playerService', [
         }
 
         videoOptions.videoMetaTitle = (video.show ? video.show.name : video.title);
-        videoOptions.videoMetaSubtitle = (video.show ? video.episodeString + ' - ' + video.name : (video.release_date ? video.release_date.substring(0, 4) : ''));
+        videoOptions.videoMetaSubtitle = (video.show ? video.episodeString + ' - ' + video.name : video.release_date.substring(0, 4));
```

Comment:
You maybe did something wrong when merging the files? Because You removed some code logic that needs to be in place. Please undo this.